libtelemetry/src/test/java/com/mapbox/android/telemetry/LocationEventTest.java

```diff
@@ -1,21 +1,35 @@
 package com.mapbox.android.telemetry;
 
+import android.app.ActivityManager;
+import android.content.Context;
+
 import org.junit.Test;
 
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertTrue;
+import static org.mockito.Mockito.RETURNS_DEEP_STUBS;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
 
 public class LocationEventTest {
 
   @Test
   public void checksLocationEvent() throws Exception {
+    Context mockedContext = mock(Context.class);
```

Comment:
This block of code is duplicated 👀 `checksLocationType` test. Could you extract it into a `private` "factory" method called for example `obtainMockedContext`?